README.md

```diff
@@ -7,11 +7,12 @@ blaa blaa here
 more blaa here
 
 ```c++
+// count to 100;
 int i = 0;
 do{
   ++i;
   std::cout << "loop count: " << i << std::endl;
-}while(true)
+}while( i < 100)
```

Comment:
I think there is no bug on counting issue.
But you missed a semicolon after while condition! Please fix.